### DISCUSSION TITLE
element picker (sbofficial.in)

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2114] [Q&A] [answered]
Firefox android stable 100.2.0 as well as nightly 
ubo 1.42.4

At this website `https://www.sbofficial.in/the-kashmir-files`
while using elment picker,when i try to pick an element, the element picker window is overzoomed
i can't see picker options like when element picker window is dragged to left/right, respective left /right picker options gets cropped

Also rules suggested seems to be cropped, screenshot attached below
![Screenshot_20220519-103032-244](https://user-images.githubusercontent.com/20338483/169222624-7ad122ca-a979-4dd4-b4dd-b33a10de088b.png)

**Comment by ghajini** [comment]
@gorhill

**Reply by gorhill** [reply]
Caused by the page's own `<meta name="viewport" ...>` tag. I don't think I can prevent the browser from scaling up the picker, but I can add a more flexible `min-width` to avoid being larger than the viewport.

**Comment by ghost** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/6fbcfc51144bdc56cd87f3a621c9d60a57c70bb4
